www/nginx/src/opnsense/www/js/nginx/src/templates/LogLine.html

```diff
@@ -0,0 +1,6 @@
+<% log_fields_vis = log_fields.filter(function(field) { %>
+<%     return field.visible === true; %>
+<% }); %>
```

Comment:
You should not do that here, you can pass functions to the template if needed when calling the template. Place that into a JS file and keep the template clean.

Reply:
also keep in mind that lodash is by far more convenient to filter lists:
https://lodash.com/docs/#filter